go.mod

```diff
@@ -3,9 +3,8 @@ module github.com/containers/skopeo
 go 1.12
 
 require (
-	github.com/containers/buildah v1.13.1 // indirect
```

Comment:
@rhatdan This part caused downgrades.

Reply:
https://github.com/containers/skopeo/pull/872

Reply:
Ok I have no idea what is going on with this.  What does buildah have to do with skopeo or containers/image?

Reply:
> What does buildah have to do with skopeo or containers/image?

https://github.com/containers/skopeo/issues/796

I’m working on making that unnecessary in the future right now, but it has to stay around at least until the next c/image release.